code/modules/antagonists/disease/disease_mob.dm

```diff
@@ -113,14 +113,14 @@ the new instance inside the host to be updated to the template's stats.
 /mob/camera/disease/say(message, bubble_type, list/spans = list(), sanitize = TRUE, datum/language/language = null, ignore_spam = FALSE, forced = null)
 	if(!message)
 		return
-	src.log_talk(message, LOG_SAY)
+	log_talk(message, LOG_SAY)
 	var/rendered = "<span class='sentientdisease'><b>[src]</b> says, \"[message]\"</span>"
-	for(var/mob/M in GLOB.mob_list)
-		if(issentientdisease(M))
-			to_chat(M, rendered)
-		else if(isobserver(M))
-			var/link = FOLLOW_LINK(M, src)
-			to_chat(M, "[link] [rendered]")
+	for(var/mob/mobs in GLOB.mob_list)
```

Comment:
This isn't good enough, no. The variable should describe its context.

For example, this could be `listener`.

You can read more here: https://github.com/tgstation/tgstation/blob/master/.github/CONTRIBUTING.md#use-descriptive-and-obvious-names

Reply:
alright thanks for the tips, i mostly copied this code from revenant's say function so i appreciate the advice